src/foam/nanos/auth/LookAndFeelService.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2019 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.INTERFACE({
+  package: 'foam.nanos.auth',
+  name: 'LookAndFeelService',
```

Comment:
Does this need to be its own service or could we just talk to a LookAndFeelDAO?

Reply:
I couldn't think of a nice way for it would work if you were accessing the DAO directly. Returning it from `find` seems weird since you're not passing in an id. Returning from `select` could work I guess if you left it up to the caller to only take the first object.

Reply:
My gut instinct is telling me that the service adds overhead to try to defer the logic on which look and feel object to the server.

If the client could just query the DAO and select the theme they want, that would also open the door to features like allowing users to select and switch between available themes if we ever wanted that support without having to add any additional logic to the service.

Basically it seems like less work to just access the DAO from the client, which also leaves options open for more flexibility later at not added cost.

Reply:
I agree. I'll remove the service and update ApplicationController to hit the DAO directly.

Reply:
Resolved by 25a838ff26f04893188bebf09972c7703bf5177e.